service/src/main/java/bio/terra/catalog/datarepo/DatarepoService.java

```diff
@@ -147,4 +148,9 @@ public SystemStatusSystems status() {
     }
     return result;
   }
+
+  public void exportDatarepoDataset(
```

Comment:
I think `exportSnapshot()` is a better name for this method as the operation isn't catalog dataset specific within the service code layer.